_data/mentors.yml

```diff
@@ -194,4 +194,13 @@
   github: hossainalikram
   image: hossain.jpg
   twitter: Hossainalikram
+  bio: 
+  
+  - name: Doris Lee
```

Comment:
@kngoledge Follow the pattern. There should not be any space before `- name` 
Will have to revert the PR.